services/scheduler/src/config/config.ts

```diff
@@ -23,6 +23,24 @@ export const appConfigSchema = Joi.object({
   dbConfig: DbConfigSchema.required(),
 }).required();
 
+export const manifestSchema = Joi.array()
+  .items(
+    Joi.object().keys({
+      url: Joi.string().required(),
+      name: Joi.string().required(),
+      actions: Joi.array().items(
+        Joi.object().keys({
+          type: Joi.string().required(),
+          http: Joi.object().keys({
+            uri: Joi.string().required(),
+            method: Joi.string().valid("GET", "POST", "PUT", "PATCH", "DELETE"),
```

Comment:
Should be required and I think that should be a full set of methods (I mean `GET, HEAD, POST, PUT, DELETE, CONNECT, OPTIONS, TRACE, PATCH`)

Reply:
I doubt we need OPTIONS, TRACE or CONNECT - we are talking about scheduling things, so you either use GET/POST/PUT/DELETE 

even HEAD is not so often, but lets allow it